django/contrib/contenttypes/migrations/0002_remove_content_type_name.py

```diff
@@ -2,8 +2,9 @@
 
 
 def add_legacy_name(apps, schema_editor):
+    alias = schema_editor.connection.alias
     ContentType = apps.get_model("contenttypes", "ContentType")
-    for ct in ContentType.objects.all():
+    for ct in ContentType.objects.using(alias).all():
```

Comment:
Minor but the trailing `.all()` is unnecessary here.

Reply:
Fair point, but it's descriptive, so I left it.

Reply:
Makes sense, this comment was mainly to make sure we are coherent with #15328 since it seems we're moving in this direction.